dev_guide/builds.adoc

```diff
@@ -1032,81 +1044,86 @@ $ oc set build-secret --source bc/sample-build sshsecret
 ----
 ====
 
-[[other-authentication]]
-===== Other
+[[trusted-certificate-authorities]]
+==== Trusted Certificate Authorities
 
-If the cloning of your application is dependent on a CA certificate,
-`.gitconfig` file, or both, then you can create a secret that contains them, add
-it to the builder service account, and then your `BuildConfig`.
+The set of TLS certificate authorities that are trusted during a `git clone`
+operation are built into the {product-title} infrastructure images.  If your Git
+server uses a self-signed certificate or one signed by an authority not trusted
+by the image, you have several options.
 
-. Create desired type of `*secret*`:
+[[create-a-secret-with-a-ca-certificate-file]]
+===== Create a secret with a CA certificate file (recommended)
+
+A secret containing a `CA certificate` in a key named
+`ca.crt` will automatically be used by `git` to trust your
+self-signed or otherwise un-trusted TLS certificate during
+the `git clone` operation. Utilizing this method is significantly
+more secure than disabling git's ssl verification which will accept
+any TLS certificate that is presented.
 
-.. To create a secret from a `.gitconfig`:
-+
 ====
 ----
-$ oc secrets new mysecret .gitconfig=path/to/.gitconfig
+# the key name ca.crt MUST be used
+$ oc secrets new mycert ca.crt=FILENAME
 ----
 ====
-.. To create a secret from a `CA certificate`:
+
+[[disable-git-tls-verification]]
+===== Disable git TLS verification
+
+You can disable git's TLS verification by setting the
+`*GIT_SSL_NO_VERIFY*` environment variable to *true* in the appropriate strategy
+section of your build configuration. You can use the
+xref:../dev_guide/environment_variables.adoc#overview[`oc set env`] command to
+manage `*BuildConfig*` environment variables.
+
+[[putting-it-all-together]]
+==== Putting it all Together 
+
+Below are several examples of how you can combine the above methods for
+creating `Authentication Secrets` for your particular needs.
```

Comment:
s/Authentication Secrets/Source Secrets/  (it's not just about auth, it's about anything related to cloning the source)